src/Component/CompositionContext/CompositionContext.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
```

Comment:
Please add some code docs what this file is about.

Reply:
solved in https://github.com/geostyler/geostyler/pull/1186/commits/587b9b93e34174a44cce9a8464bcb50512687670